Client/Frontend/Browser/Tabs/TabManagerMiddleware.swift

```diff
@@ -4,16 +4,14 @@
 
 import Common
 import Redux
+import TabDataStore
 
 class TabManagerMiddleware {
-    var tabManager: TabManager
+    // TODO: [7863] Part of ongoing WIP for Redux + iPad Multi-window.
+    var tabManagers: [SceneUUID: TabManager] = [:]
```

Comment:
@OrlaM and @mattreaganmozilla for the coordination between the two projects I will continue to use `defaultTabManager` for the tab tray refactor like used in this PR, and also check-in if any other change is needed on my side to support the early work for iPad Multi-window.